challenges/array-binary-search/array-binary-search.js

```diff
@@ -0,0 +1,12 @@
+const arrayBinarySearch = (array, searchKey) => {
```

Comment:
not really a binary search, but it looks good.

Binary search adds a bit of efficiency when dealing with already sorted arrays.